modules/find_assembly/main.nf

```diff
@@ -9,6 +9,8 @@ process FIND_PRIMARY_ASSEMBLY {
     path "*.not_linked.tsv", emit: not_linked_mags
 
     script:
+    def cleanup_flag = params.cleanup ? "--cleanup" : ""
+    def debug_flag = params.debug ? "--debug" : ""
```

Comment:
The most nf-core like way to do it would provided this through the modules.config as args, and then pass as ${args}.
You don't have to do that here, just saying ;)

Reply:
Thanks for teaching me, senpai 🌸